grammars/fortran - punchcard.cson

```diff
@@ -1,4 +1,6 @@
 'comment': '?i: has to be added everywhere because fortran is case insensitive; NB: order of matching matters'
+'name': 'Fortran - Punchcard'
+'scopeName': 'source.fortran'
 'fileTypes': [
   'f'
```

Comment:
If I understand correctly, you will consider f extensions as modern Fortran?
What about letting it to the new punchcard fortran style?
Maybe I'm wrong about the use people have of .f files.